rbac/server/db/db_utils.py

```diff
@@ -69,3 +70,8 @@
             count += 1
             time.sleep(delay)
     return resource_found
+
+
+def sanitize_query(query):
+    """Remove special characters from a given query."""
```

Comment:
Args should be defined per our agreed on pattern:
"""function description

Args:
    query:
         str:  word to search
"""

Reply:
Updated.